### PR TITLE
I797 daysim for demand2

### PR DESCRIPTION
**Pull request by gabriel-happle**
enable switch between arcgis and daysim radiation as input files for the demand calculation.

Documentation and testing still in progress

### CODE REVIEW

**Comment by daren-thomas** [comment]
hey @gabriel-happle - do you need help with this?

**Comment by JIMENOFONSECA** [comment]
waiting for #908 to continue testing.

File: cea/default.config [hunk 1]
```diff
@@ -146,6 +146,10 @@ use-dynamic-infiltration-calculation = false
 use-dynamic-infiltration-calculation.type = BooleanParameter
 use-dynamic-infiltration-calculation.help = Use the dynamic infiltration calculation (slower!)
 
+use-daysim-radiation = true
+use-daysim-radiation.type = BooleanParameter
+use-daysim-radiation.help = set to false to use ArcGIS radiation as input, default is true
```
Comment (daren-thomas) [inline cea/default.config]: Check the ArcGIS interface for the demand tool to see if the help message makes sense. Maybe it needs some reformulation?

File: cea/demand/demand_main.py [hunk 2]
```diff
@@ -15,7 +15,7 @@
 import cea.config
 from cea.demand import occupancy_model
 from cea.demand import thermal_loads
-from cea.demand.thermal_loads import BuildingProperties
+from cea.demand.building_properties import BuildingProperties
```
Comment (daren-thomas) [inline cea/demand/demand_main.py]: i *love* this!

File: cea/demand/sensible_loads.py [hunk 3]
```diff
@@ -89,9 +88,12 @@ def calc_I_sol(t, bpr, tsd, gv):
         I_rad: vector solar radiation re-irradiated to the sky.
     """
 
-    Asol_wall, Asol_roof, Asol_win = calc_Asol(t, bpr, gv)
+    # calc re-radiatioin
```
Comment (daren-thomas) [inline cea/demand/sensible_loads.py]: # calc re-radiation

File: cea/demand/sensible_loads.py [hunk 4]
```diff
@@ -120,7 +122,7 @@ def calc_I_rad(t, tsd, bpr, Rse):
     I_rad_roof = Rse * bpr.rc_model['U_roof'] * calc_hr(bpr.architecture.e_roof, theta_ss) * bpr.rc_model[
         'Aroof'] * theta_ss
     I_rad_wall = Rse * bpr.rc_model['U_wall'] * calc_hr(bpr.architecture.e_wall, theta_ss) * bpr.rc_model[
-        'Awall_all'] * theta_ss
+        'Aop_sup'] * theta_ss
```
Comment (daren-thomas) [inline cea/demand/sensible_loads.py]: is this a bugfix?

Comment (JIMENOFONSECA) [reply cea/demand/sensible_loads.py]: yep

File: cea/demand/demand_main.py (hunk 2 above)
Comment (JIMENOFONSECA) [reply cea/demand/demand_main.py]: :-)

File: cea/demand/sensible_loads.py (hunk 3 above)
Comment (JIMENOFONSECA) [reply cea/demand/sensible_loads.py]: fixed

**Comment by JIMENOFONSECA** [comment]
@daren-thomas the tests should pass now

File: cea/default.config (hunk 1 above)
Comment (JIMENOFONSECA) [reply cea/default.config]: yep

**Comment by JIMENOFONSECA** [comment]
@daren-thomas would you mind accepting the review so we can merge?

**Comment by JIMENOFONSECA** [comment]
@daren-thomas btw about this test that does not pass, is there a way to automatically regenerate those values of the calc_thermal loads? it is frustrating having to do that by hand.

**Comment by daren-thomas** [comment]
@JIMENOFONSECA I'm on it!

BTW: Yes - see the documentation here: http://city-energy-analyst.readthedocs.io/en/latest/how-to-test-the-cea.html#types-of-tests-performed

**Pull request merged**